Core/src/main/java/com/plotsquared/core/command/FlagCommand.java

```diff
@@ -270,14 +270,14 @@ public Collection<Command> tab(
                     .filter(value -> value.startsWith(args[0].toLowerCase(Locale.ENGLISH)))
                     .map(value -> new Command(null, false, value, "", RequiredType.NONE, null) {
                     }).collect(Collectors.toList());
-        } else if (Arrays.asList("set", "add", "remove", "delete", "info")
+        } else if (Arrays.asList("set", "s", "add", "a", "remove", "r", "delete", "info", "i")
```

Comment:
Yeah I'd like to see these extracted to static final `Collections.immutableSet` at the class level - this would be best practices for Java